.circleci/config.templ.yml

```diff
@@ -141,12 +141,22 @@ commands:
             - start_docker_services:
                 env: SNAPSHOT_CI=1
                 services: testagent << parameters.docker_services >>
+            - run: ulimit -c unlimited
```

Comment:
I left a top-level comment in the conversation, but is this generally respected?  There are lots of ways to interfere with corefile generation, and modern Linux distros basically try them all...